views/templates/products/index.hbs

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+
+<body>
+  {{> header}}
+
+  {{#each products}}
+  <div>
+    <h2>{{name}}</h2>
```

Comment:
You probably don't want `h2` tags here. Semantic HTML says that should only be a header for a section where these should probably be `div` elements with CSS controlling font styling like boldness and layout.